### ISSUE TITLE
No way to pass Shelf objects to Robots

### ISSUE THREAD

**Issue by TheGoldenMallet** [theliamcrawford/Warehouse---CS_2820#11]
There's currently no way for a robot to be passed a Shelf object once they reach their target location.

Currently when a robot reaches it's target location, it attempts to call Floor.getShelfAt(Point p), however Floor doesn't appear to have the shelves. A class needs to be updated to return the Shelf object at a location when called.

Possible solutions:
Floor can return the Shelf object via a method "getShelfAt(Point p)"
Inventory can return the Shelf object via a method "getShelfAt(Point p)"

**Comment by theliamcrawford** [comment]
I'm thinking the getShelfAt(Point p) should be added to the Floor, but would that mean that the floor is now a tickable object? Or is "tick" still irrelevant to the floor?

**Comment by TheGoldenMallet** [comment]
It wouldn't need it's own tick so long as it updates it's records when RS calls it. Then the action would be triggered when necessary by RS.tick()

**Comment by TheGoldenMallet** [comment]
After discussion in class we've decided we'll do the following:

**Floor** will track shelf objects and pass them to RobotScheduler. Floor will need:
-grabShelf(Point p)
-setShelf(Shelf s)
or something similar.

**Master** will need to be updated to pass ArrayList<Integer> of shelf numbers when passing order information, or pass Integer shelf numbers for restocking. 

I have updated RobotScheduler to work with these changes.
